tests/test_key_standardisation.cc

```diff
@@ -25,10 +25,11 @@
  * Boston, MA 02111-1307, USA.
  */
 
-#include "tests.hh"
-#include "utils/key_standardisation.hh"
+#include <boost/test/unit_test.hpp>
```

Comment:
ours before external?

see [here](https://github.com/cosmo-epfl/librascal/blob/master/docs/source/dev_guide/coding-convention.rst#names-and-order-of-includes)

Reply:
ok, missed this. I would rather change the style guide to follow google's, and enforce headers with `include-what-you-use`, but this should be for a separate PR.

Reply:
Also, this should be part of clang-format or at the very least cpplint, instead of wasting reviewer and implementation time.

Reply:
Done in a6316c0ae6950ae77c4eb78b93c8629faf374709